vnext/JSI/Universal/JSI.Universal.vcxproj

```diff
@@ -90,11 +90,6 @@
       <Optimization>Disabled</Optimization>
     </ClCompile>
   </ItemDefinitionGroup>
-  <ItemDefinitionGroup Condition="'$(Configuration)'=='WinUI3'">
-    <ClCompile>
-      <Optimization>Disabled</Optimization>
-    </ClCompile>
-  </ItemDefinitionGroup>
```

Comment:
This is definitely kinda suspect, but I'm curious if the history behind it. @asklar do you know why this was here?

Reply:
huh maybe I missed removing this when I switched it from being a config to being a property. I think it should be harmless to remove these.